web-server/django_web_server/website/server.py

```diff
@@ -46,6 +49,60 @@ def getClients(self):
             print("Server issues" + str(e))
             return []
 
+    def getStatistics(self):
```

Comment:
The code has some inconsistencies in variable and function naming. Should this, for example, be changed to `get_statistics`, so that it is PEP-8 compliant?